content/en/Platform Deep Dive/Collaboration/Organization/manage-users.md

```diff
@@ -53,42 +53,32 @@ To remove a user from your organization:
+On the **Pentest Collaborators** tab of the **People** page, you can see users who don't belong to your organization but can collaborate on specific pentests. This includes:
 
-{{% owner-member-team-member %}}
+- [Pentest Team Members](/getting-started/glossary/#pentest-team-member)
+- [Pentesters](/getting-started/glossary/#pentester)
+- [Pentest Leads](/getting-started/glossary/#pentest-lead)
 
-### Add a Pentest Team Member
-
-To add a Pentest Team Member to a specific pentest:
-
-1. In the Cobalt app, select **Pentests**, and locate the desired pentest.
-1. On the pentest page, select **Settings** > **Collaborators**.
-1. Enter the user's email address, and select **Add Team Member**.
-
-The user gets an invitation to collaborate on the pentest.
-
-![Add a Pentest Team Member to a pentest](/deepdive/AddPentestTeamMember.png "Add a Pentest Team Member to a pentest")
-
-### Remove a Pentest Team Member
+To view pentests on which the user collaborates, select the link under **Pentests**. From here, you can navigate to the pentest page.
 
-To remove a Pentest Team Member from a pentest:
-
-1. Navigate to the desired pentest in one of the following ways:
-   - Select **Pentests**, and locate the pentest.
-   - On the **People** page, locate the desired user, select **# Pentests**, and select the pentest from which you want to remove the user.
-1. On the pentest page, select **Settings** > **Collaborators**.
-1. Locate the user you want to remove, select **Remove User** ![Trash bin icon](/icons/Trash.png "Trash bin icon"), and select **Yes** in the overlay to confirm.
+Pentest collaborators who don't belong to the organization can [manage pentest collaborators](/platform-deep-dive/collaboration/manage-collaborators/) for a specific pentest.
 
 ### Remove a User from All Pentests
 
-As an Organization Owner, you can remove a Pentest Team Member—who doesn't belong to the organization—from all pentests they collaborate on.
+{{% remove-from-all-pentests %}}
 
 1. In the Cobalt app, select your organization in the upper-left corner of the screen.
 1. Navigate to the **People** page > **Pentest Collaborators**.
 1. For the Team Member you want to remove, select the three-dot icon {{% three-dots %}} > **Remove User**, and select **Yes** in the overlay to confirm.
 
 ![Remove a Pentest Team Member from all pentests](/deepdive/RemoveCollaboratorFromAllPentests.png "Add a Pentest Team Member from all pentests")
 
-To view the pentests a Pentest Team Member collaborates on, select the link under **Pentests**. From here, you can navigate to the pentest page.
+### Add a Pentest Team Member
+
+As a pentest collaborator, you can [add users to a specific pentest](/platform-deep-dive/collaboration/manage-collaborators/#add-a-pentest-team-member). This doesn't require the Organization Owner role.
```

Comment:
As a pentest team member vs As a pentest collaborator. 

Not all pentest collaborators from the org page will be able to add users. On the people > pt collaborators page, we include pentesters. Pentesters won't be able to add other folks to a pentest

Same comment within the Remove section.